### ISSUE TITLE
on error how to scroll page to input box ?

### ISSUE THREAD

**Issue by yogeshsaroya** [1000hz/bootstrap-validator#540]
Hi 

on validation error, how to scroll page to input box ?

**Comment by 1000hz** [comment]
That should happen automatically since the `focus` option defaults to true. See http://1000hz.github.io/bootstrap-validator/#validator-options

**Comment by yogeshsaroya** [comment]
It;s not working with form submit via ajax.

**Comment by yogeshsaroya** [comment]
I have used latest js file, now working. 
How to use "$.fn.validator.Constructor.FOCUS_OFFSET"  ?

**Comment by 1000hz** [comment]
You only need to use it if you need to adjust the padding in case you have something like a fixed navbar or something.

You'd use it like

`$.fn.validator.Constructor.FOCUS_OFFSET = 70`

Where 70 is the number of pixels from the top of the screen you want the input to be scrolled to.

**Comment by yogeshsaroya** [comment]
I used latest js, then focus is working fine.

for fixed navbar i used this code. 

	$('#Form').validator().on('submit', function (e) {
	    if (e.isDefaultPrevented()) {
	        // handle the invalid form...

	        $('html, body').animate({ scrollTop: $(".has-error:first :input").offset().top - 100 }, 500);
	        $(".has-error:first :input").focus();

	    } else {
	      
	    }
	});

Thanks for your time.
